server/mdm/android/tests/enterprise/enterprise_test.go

```diff
@@ -50,3 +62,40 @@ func (s *enterpriseTestSuite) TestGetEnterprise() {
 	s.Do("DELETE", "/api/v1/fleet/android_enterprise", nil, http.StatusOK)
 	s.DoJSON("GET", "/api/v1/fleet/android_enterprise", nil, http.StatusNotFound, &resp)
 }
+
+func (s *enterpriseTestSuite) TestEnterpriseSSE() {
+	s.SetupTest()
+
+	// Test happy path
+	resp := s.Do("GET", "/api/v1/fleet/android_enterprise/signup_sse", nil, http.StatusOK)
+	sseDone := make(chan struct{})
+	buf := make([]byte, 1024)
+	go func() {
+		n, _ := resp.Body.Read(buf)
+		assert.Equal(s.T(), service.SignupSSESuccess, string(buf[:n]))
+		close(sseDone)
+	}()
+
+	time.Sleep(50 * time.Millisecond)
+	s.AppConfig.MDM.AndroidEnabledAndConfigured = true
+
+	select {
+	case <-sseDone:
+		s.T().Log("SSE done")
+	case <-time.After(2 * time.Second):
+		s.T().Fatal("Timed out waiting for SSE")
+	}
+
+	// Test with Android already enabled
+	resp = s.Do("GET", "/api/v1/fleet/android_enterprise/signup_sse", nil, http.StatusOK)
+	n, _ := resp.Body.Read(buf)
```

Comment:
You should probably do `require.NoError(t, err)` instead of ignoring the error here.